docs/notebooks/dispersion/hollweg_dispersion.ipynb

```diff
@@ -0,0 +1,792 @@
+    "    params1[\"L\"] * ky,\n",
+    "    params2[\"L\"] * ky,\n",
+    "    params3[\"L\"] * ky,\n",
+    "]\n",
+    "big_omega = [\n",
+    "    abs(omegas0[\"alfven_mode\"] / (params0[\"va\"] * kz)),\n",
+    "    abs(omegas1[\"alfven_mode\"] / (params1[\"va\"] * kz)),\n",
+    "    abs(omegas2[\"alfven_mode\"] / (params2[\"va\"] * kz)),\n",
+    "    abs(omegas3[\"alfven_mode\"] / (params3[\"va\"] * kz)),\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fs = 14  # default font size\n",
+    "figwidth, figheight = plt.rcParams[\"figure.figsize\"]\n",
+    "figheight = 1.6 * figheight\n",
+    "fig = plt.figure(figsize=[figwidth, figheight])\n",
+    "\n",
+    "# define colormap\n",
+    "cmap = plt.get_cmap(\"viridis\")\n",
+    "slicedCM = cmap(np.linspace(0, 0.6, 4))\n",
+    "\n",
+    "# plot\n",
+    "plt.plot(\n",
+    "    k_prime[0],\n",
+    "    big_omega[0],\n",
+    "    c=slicedCM[0],\n",
+    "    ms=1,\n",
+    ")\n",
+    "ax = plt.gca()\n",
+    "ax.plot(\n",
+    "    k_prime[1],\n",
+    "    big_omega[1],\n",
+    "    c=slicedCM[1],\n",
+    "    ms=1,\n",
+    ")\n",
+    "ax.plot(\n",
+    "    k_prime[2],\n",
+    "    big_omega[2],\n",
+    "    c=slicedCM[2],\n",
+    "    ms=1,\n",
+    ")\n",
+    "ax.plot(\n",
+    "    k_prime[3],\n",
+    "    big_omega[3],\n",
+    "    c=slicedCM[3],\n",
+    "    ms=1,\n",
+    ")\n",
+    "\n",
+    "# adjust axes\n",
+    "ax.set_xlabel(r\"$k_{y}L$\", fontsize=fs)\n",
+    "ax.set_ylabel(r\"$|\\Omega|$\", fontsize=fs)\n",
+    "ax.set_yscale(\"linear\")\n",
+    "ax.set_xscale(\"linear\")\n",
+    "ax.set_xlim(0, 1.5)\n",
+    "ax.set_ylim(0.96, 1.8)\n",
+    "ax.tick_params(\n",
+    "    which=\"both\",\n",
+    "    direction=\"in\",\n",
+    "    width=1,\n",
+    "    labelsize=fs,\n",
+    "    right=True,\n",
+    "    length=5,\n",
+    ")\n",
+    "\n",
+    "# add labels for beta\n",
+    "plt.text(1.51, 1.75, \"1/20$=\\\\beta$\", c=slicedCM[0], fontsize=fs)\n",
+    "plt.text(1.51, 1.63, \"1/2\", c=slicedCM[1], fontsize=fs)\n",
+    "plt.text(1.51, 1.44, \"2\", c=slicedCM[2], fontsize=fs)\n",
+    "plt.text(1.51, 0.97, \"1/2000\", c=slicedCM[3], fontsize=fs)\n",
+    "\n",
+    "# plot caption\n",
+    "txt = (\n",
+    "    \"Fig. 2 Hollweg 1999 reproduction. The Alfvén mode is plotted for an electron\\n\"\n",
+    "    \"-proton plasma with $\\\\beta=$ 1/20, 1/2, 2, 1/2000, $|\\\\Omega|=|\\\\omega / k_{z} v_{A}|$, and $L=c_{s}$ $/ |\\\\omega_{ci}|$.\\n\"\n",
+    ")\n",
+    "\n",
+    "plt.figtext(-0.08, -0.21, txt, ha=\"left\", transform=ax.transAxes, fontsize=16);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note: Hollweg takes $k_{\\perp}=k_{y}$ for k propagating in the $y$-$z$\n",
+    "plane as in the plot above. Contrarily, Bellan takes $k_{\\perp}=k_{x}$\n",
+    "for k propagating in the $x$-$z$ plane."
```

Comment:
```markdown
<div class="alert alert-info">
Note

Hollweg takes $k_{\perp}=k_{y}$ for k propagating in the $y$-$z$
plane as in the plot above. Contrarily, Bellan takes $k_{\perp}=k_{x}$
for k propagating in the $x$-$z$ plane.
</div>
```

No need to change the wording, but I'm thinking it'd be good to use the same note syntax as we used above in the beginning of the notebook.